tfhe/src/integer/server_key/radix/mod.rs

```diff
@@ -1,6 +1,7 @@
 mod add;
 mod bitwise_op;
 mod comparison;
+mod modulus_switch_compression;
```

Comment:
the primitives are parallelized right ? so :/

let's move the module and have the primitive have the parallelized suffix, ok to not provide the sequential version, but we need consistency in the API, I think I mentionned the naming change

Reply:
Good catch